test/info.js

```diff
@@ -66,6 +66,7 @@ describe('Info operations', () => {
     it('should work with only metadata', () => {
       const input = {
         metadata: {
+          id: 15,
```

Comment:
this is the wrong test. we shouldn't be using metadata here. we should be respecting what comes in through `req.params.query`